01.elements/mediapro/02.advanced/04.cloud-front/docs.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Integration with Cloud Front 
+taxonomy:
+    category: docs
+---
+
+## How To Integrate Media Pro with Cloud Front
+
+[CloudFront](https://aws.amazon.com/cloudfront/) Integration is available for JWPlayer and MediaElement players. To enable the integration check **CloudFront Integration** option on the element render settings. 
+ 
+| Field       | Description |
+| :---------- | :---------- |
+| **Distribution** | Set the CloudFront Distribution Domain Name.<br>Ex: `'s1u4jtfxixe2o5'` without `.cloudfront.net.`
```

Comment:
Add some information where this number should be retrieved: "From Amazon CloudFront configuration"